io.typefox.yang.parent/io.typefox.yang/src/test/java/io/typefox/yang/tests/AbstractYangTest.xtend

```diff
@@ -39,10 +40,26 @@ abstract class AbstractLinkingTest {
 		validator.assertError(obj, obj.eClass, code)
 	}
 	
+	protected def assertError(EObject obj, String code, int offset, int length, String... messageParts) {
```

Comment:
Dealing with offset and length in expectations is cumbersome for the reader. 
Could we use a string instead, that is searched in the text (first match)?